docs/querying/set-query-context.md

```diff
@@ -0,0 +1,310 @@
+  }'
+```
+
+You can also combine `SET` statements with the `context` field. If you include both, the parameter value in SET takes precedence:
+
+```json
+{
+  "query": "SET timeout = 90000; SELECT COUNT(*) FROM data_source",
+  "context": {
+    "timeout": 30000,  // This will be overridden by SET
+    "priority": 100    // This will still apply
+  }
+}
+```
+
+`SET` statements only apply to the query in the same request. Subsequent requests are not affected. 
+
+`SET` statements work with `SELECT`, `INSERT`, and `REPLACE` queries.
+
+For more details on how to use the `SET` command in your SQL query, see [SET](sql.md#set).
+
+:::info
+ You cannot use `SET` statements when using Druid SQL JDBC connections.
+:::
+
+
+## Native queries
+
+For native queries, you can include query context parameters in a JSON object named `context` within your query structure.
+
+Native queries in Druid are JSON objects that you typically send to the Broker or Router processes. You can submit native queries in two ways:
+
+**Using curl with a query file:**
+
+First, you need to create a JSON file with your query and context parameters. The following example is the content from `query.json` file.
+
+```json
+{
+  "queryType": "timeseries",
+  "dataSource": "sample_datasource",
+  "granularity": "day",
+  "descending": "true",
+  "filter": {
+    "type": "and",
+    "fields": [
+      { "type": "selector", "dimension": "sample_dimension1", "value": "sample_value1" },
+      { "type": "or",
+        "fields": [
+          { "type": "selector", "dimension": "sample_dimension2", "value": "sample_value2" },
+          { "type": "selector", "dimension": "sample_dimension3", "value": "sample_value3" }
+        ]
+      }
+    ]
+  },
+  "aggregations": [
+    { "type": "longSum", "name": "sample_name1", "fieldName": "sample_fieldName1" },
+    { "type": "doubleSum", "name": "sample_name2", "fieldName": "sample_fieldName2" }
+  ],
+  "postAggregations": [
+    { "type": "arithmetic",
+      "name": "sample_divide",
+      "fn": "/",
+      "fields": [
+        { "type": "fieldAccess", "name": "postAgg__sample_name1", "fieldName": "sample_name1" },
+        { "type": "fieldAccess", "name": "postAgg__sample_name2", "fieldName": "sample_name2" }
+      ]
+    }
+  ],
+  "intervals": [ "2012-01-01T00:00:00.000/2012-01-03T00:00:00.000" ],
+  // Add context parameters here
+  "context": {
+    "timeout": 30000,        // Query timeout in milliseconds
+    "priority": 100,         // Higher priority queries get more resources
+    "useCache": false,       // Disable cache for testing
+    "finalize": true         // Finalize aggregation results
+  }
+}
+```
+
+Then you can submit the query using curl:
+
+```bash
+# Replace localhost:8888 with your Druid Broker or Router address
+curl -X POST 'http://localhost:8888/druid/v2/?pretty' \
+  -H 'Content-Type:application/json' \
+  -H 'Accept:application/json' \
+  -d @query.json
+```
+
+**Using the web console:**
```

Comment:
Consolidate with earlier web console section (don't create a separate section here), although you can link to it. Are you able to set parameters in the **Edit context** option in the web console? What happens if you set parameters in both areas (in the console and in the query itself).

Notice that this section predominantly describes "how to send native queries in the web console" which is not in scope of this doc.